google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/CloudDebugProcess.java

```diff
@@ -261,9 +261,12 @@ public void run(@NotNull ProgressIndicator indicator) {
     // Start breakpoints refresh job on first use.
     getStateController().addListener(this);
     getStateController().startBackgroundListening();
-    SwingUtilities.invokeLater(new Runnable() {
-      @Override
-      public void run() {
+    SwingUtilities.invokeLater(() -> {
+      // If a breakpoint handler has not been created before, getBreakpointHandler() will attempt to
+      // create one. But if the project is disposed of, this creation will fail because it will try
+      // to access the project's components when creating the PsiManager. This guards against a
+      // failure in this scenario.
+      if (breakpointHandlers != null || !getXDebugSession().getProject().isDisposed()) {
```

Comment:
Actually this seems to have been the trigger: https://github.com/JetBrains/intellij-community/commit/95a2625ec31ae0327ccf962c31e18ace2301e3e0

Now the events stacked up in the EDT aren't processed until after the project is disposed. We could manually call this somewhere in the test instead, what do you think?